.travis.yml

```diff
@@ -1,7 +1,23 @@
-sudo: false
 language: python
-python:
-  - "3.5"
+python: 3.6
+matrix:
+  include:
+    - python: 3.7
+      dist: xenial
+      sudo: true
+      env: TOXENV=py37-dj20,py37-dj21,flake8
```

Comment:
This all looks great! I have 2 suggestions:

1. I don't think we need to run flake8 in every environment. My understanding is that it does the same syntax checks regardless of python version? Maybe we only need one for Py 3.x and one Py 2.x? Please let me know if I'm wrong.
2. We could look at using [tox-travis](https://github.com/tox-dev/tox-travis) to simplify the config, but since you've already done the config we probably don't need to. Something to consider for future perhaps.

Reply:
>     1. I don't think we need to run flake8 in every environment. My understanding is that it does the same syntax checks regardless of python version? Maybe we only need one for Py 3.x and one Py 2.x? Please let me know if I'm wrong.

This is probably too much for this project but I generally find this approach to be good. If there are syntax changes on the python level between versions, running flake on each version will catch them. We can got either way though.



>     2\. We could look at using [tox-travis](https://github.com/tox-dev/tox-travis) to simplify the config, but since you've already done the config we probably don't need to. Something to consider for future perhaps.

I didn't know about that, thanks! While tox-travis is way cleaner compated to what i've done, i wonder if it can support 3.7 and it's custom conf.

Reply:
> I didn't know about that, thanks! While tox-travis is way cleaner compated to what i've done, i wonder if it can support 3.7 and it's custom conf.

Not sure. And not worth holding this up to find out. Possibly worth exploring in future though. Great work!